### ISSUE TITLE
Update Terra Classic Development and Validator Documentation

### ISSUE THREAD

**Issue by LuncBurner** [classic-terra/core#81]
**Problem definition**
The Terra Classic documentation on https://classic-docs.terra.money/ is out of date and needs to be updated.

**Additional context**
- [ ] Update https://classic-docs.terra.money/ with updated documentation.

**Comment by ZaradarBH** [comment]
@shackra maybe this is something you want to lend us a hand with? :)

**Comment by shackra** [comment]
yeah, I can probably do this too! what should I refer to as source of truth to judge what's out of date, @ZaradarBH ?

**Comment by ZaradarBH** [comment]
> yeah, I can probably do this too! what should I refer to as source of truth to judge what's out of date, @ZaradarBH ?

Ping me on slack and we can have a VOIP session about it :)

**Comment by nghuyenthevinh2000** [comment]
@blendsdk is it done yet?
